sansa-ml-spark/src/main/scala/net/sansa_stack/ml/spark/clustering/algorithms/PIC.scala

```diff
@@ -1,14 +1,80 @@
 package net.sansa_stack.ml.spark.clustering.algorithms
 
+import com.typesafe.config.ConfigFactory
+import org.apache.jena.graph.{ NodeFactory, Triple }
 import org.apache.spark.graphx.Edge
 import org.apache.spark.graphx.Graph
 import org.apache.spark.mllib.clustering.PowerIterationClustering
-import org.apache.spark.rdd._
-import org.apache.spark.sql._
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.SparkSession
+import scala.collection.mutable.{ ArrayBuffer, HashMap }
 
+import net.sansa_stack.ml.spark.clustering.datatypes.Distance
+import net.sansa_stack.ml.spark.clustering.utils._
 
-class PIC {
+class PIC(input: RDD[Triple]) extends Serializable with ClusterAlgo {
+  val prefixID = "<http://clustering.slipo.eu/poi/"
```

Comment:
This is quite static, we should find the way to read the prefixes from a file (e.g. a ttl file) in the future.

Reply:
Yes, it needs to change. Maybe, we can add these static in the application.properties file.